samples/model-viewer/src/main/java/io/github/sceneview/sample/modelviewer/MainFragment.kt

```diff
@@ -6,59 +6,39 @@ import androidx.core.view.isGone
 import androidx.fragment.app.Fragment
 import androidx.lifecycle.lifecycleScope
 import io.github.sceneview.SceneView
-import io.github.sceneview.loaders.loadHdrIndirectLight
-import io.github.sceneview.loaders.loadHdrSkybox
 import io.github.sceneview.math.Position
 import io.github.sceneview.math.Rotation
-import io.github.sceneview.nodes.ModelNode
-import io.github.sceneview.nodes.ViewNode
+import io.github.sceneview.math.Scale
+import io.github.sceneview.node.ModelNode
+import kotlinx.coroutines.launch
 
 class MainFragment : Fragment(R.layout.fragment_main) {
 
-    lateinit var sceneView: SceneView
-    lateinit var loadingView: View
+    private lateinit var sceneView: SceneView
+    private lateinit var loadingView: View
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        sceneView = view.findViewById<SceneView>(R.id.sceneView).apply {
-            setLifecycle(lifecycle)
-        }
+        sceneView = view.findViewById(R.id.sceneView)
         loadingView = view.findViewById(R.id.loadingView)
 
-        lifecycleScope.launchWhenCreated {
+        viewLifecycleOwner.lifecycleScope.launch {
             val hdrFile = "environments/studio_small_09_2k.hdr"
-            sceneView.loadHdrIndirectLight(hdrFile, specularFilter = true) {
-                intensity(30_000f)
-            }
-            sceneView.loadHdrSkybox(hdrFile) {
-                intensity(50_000f)
-            }
+            sceneView.environmentLoader.loadHDREnvironment(hdrFile)
+            val modelFile = "models/MaterialSuite.glb"
+            val modelInstance = sceneView.modelLoader.createModelInstance(modelFile)
 
-            val model = sceneView.modelLoader.loadModel("models/MaterialSuite.glb")!!
-            val modelNode = ModelNode(sceneView, model).apply {
+            val modelNode = ModelNode(
+                modelInstance = modelInstance,
+                scaleToUnits = 2.0f,
+            ).apply {
                 transform(
                     position = Position(z = -4.0f),
                     rotation = Rotation(x = 15.0f)
                 )
```

Comment:
This is not the expected result? 


![1](https://github.com/SceneView/sceneview-android/assets/7093480/50b40f81-8958-471a-800c-178a7f34008c)